frontend/javascripts/test/shaders/shader_syntax.spec.ts

```diff
@@ -22,8 +22,24 @@ test("Shader syntax: Ortho Mode", (t: ExecutionContext<any>) => {
     globalLayerCount: 2,
```

Comment:
All these tests have the same / similar `textureLayerInfos` maybe having different settings in some tests might be nice. 

Although this is not a must, as this should also be covered by the e2e tests